src/QMCTools/CMakeLists.txt

```diff
@@ -71,3 +71,6 @@ FOREACH(p ${QTOOLS})
 
 ENDFOREACH(p ${QTOOLS})
 
+#ppconvert
+SUBDIRS(ppconvert)
```

Comment:
Could you put this under a flag, such as BUILD_PPCONVERT.  I'm not sure we want to start building ppconvert on every platform by default.

Reply:
See ppconvert/src/CMakeLists.txt  EXCLUDE_FROM_ALL means it is not built by default on any platform.